### PR TITLE
Add TrueNAS Scale guide

### PR DESCRIPTION
**Pull request by TheIceCreamTroll**

### CODE REVIEW

**Comment by majora2007** [comment]
I missed this PR from you, I'll take a look and get back to you.

File: pages/02.install/06.truenas-scale/default.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+As there currently isn't a TrueCharts app for Kavita's nightly releases, it needs to be installed as a Docker image.
+
+Go to the Available Applications page and click "Launch Docker Image".
+
+Modify the following sections:
+- Application Name
+  - Set the application name to whatever you want.
+- Container Images
+  - Set the image repository to `kizaing/kavita`.
+  - Set the image tag to `nightly`.
```
Comment (majora2007) [inline pages/02.install/06.truenas-scale/default.md:36]: This should be `latest`. We don't want users to be on nightly by default, as that's for beta testers and can be unstable.

**Review by majora2007 — APPROVED** [review]
Overall looks good. I can merge this and take care of the suggestion for you.

**Comment by majora2007** [comment]
Looks like there are some conflicts, so if you could please fix them, then we can merge this awesome addition.

File: pages/02.install/06.truenas-scale/default.md (hunk 1 above)
Comment (TheIceCreamTroll) [reply pages/02.install/06.truenas-scale/default.md:36]: The only reason to chose the "Launch Docker Image" method over the official TrueCharts app is to use the nightly release, hence why I made it the default tag and called the section "Kavita (Nightly)".

If you are against users using `nightly` by default, perhaps I could add a disclaimer that says unless they want to test the latest changes, users should stick to the TrueCharts method. How does this sound?

Comment (majora2007) [reply pages/02.install/06.truenas-scale/default.md:36]: Ahh...I missed the two different sections. Yeah, I would put an additional disclaimer underneath to inform the users this is a version for beta testers and there may be bugs. Maybe linking to the discord dev-branch-testing (https://discord.com/channels/821879810934439936/851515194185744464) would be a good idea as well.

**Comment by TheIceCreamTroll** [comment]
Still need to add the note about nightly being for beta testers and a link to the dev-branch channel, but I can put that in a new PR.

**Comment by majora2007** [comment]
New PR would be good. Sorry about that.

**Pull request merged**